net/getting-started/licensing/_index.md

```diff
@@ -6,98 +6,129 @@ url: /net/licensing/
+  * The folder containing the entry assembly (your .exe)
+  * An embedded resource in the assembly that called the dll of the component (included in Aspose.Slides).
+* To avoid the limitations associated with the evaluation version, you need to set a license before using Aspose.Slides. You only have to set a license once per application or process.
 
-* To avoid the limitations associated with the evaluation version, you need to set a license before using **Aspose.Slides**. You are only required to set a license once per application or process.
+{{% alert color="primary" %}} 
 
+You may want to see [Metered Licensing](/slides/net/metered-licensing/).
 
-## **Applying a License**
-In Aspose.Slides for .NET, a license can be loaded from a **file**, **stream**, or **embedded resource**. 
+{{% /alert %}} 
 
-Aspose.Slides for .NET typically tries to find the license in these locations:
 
-* An explicit path
-* The folder containing the dll of the component (included in Aspose.Slides)
-* The folder containing the assembly that called the dll of the component (included in Aspose.Slides)
-* The folder containing the entry assembly (your .exe)
-* An embedded resource in the assembly that called the dll of the component (included in Aspose.Slides)
+## **Applying a License**
+A license can be loaded from a **file**, **stream**, or **embedded resource**. 
 
 {{% alert color="primary" %}}
 
-For license operations, Aspose.Slides provides the [License](https://reference.aspose.com/slides/net/aspose.slides/license) class. 
+Aspose.Slides provides the [License](https://reference.aspose.com/slides/net/aspose.slides/license) class for licensing operations.
 
 {{% /alert %}} 
 
-### **Using a File**
+### **File**
 The easiest method of setting a license requires you to place the license file in the same folder containing the component's DLL (included in Aspose.Slides) and specify the file name without its path.
 
 This C# code shows you how to set a license file:
 
 ``` csharp
-
- //Instantiate an instance of license and set the license file through its path
+// Instantiates the License class 
 Aspose.Slides.License license = new Aspose.Slides.License();
-license.SetLicense("Aspose.Slides.lic");
 
+// Sets the license file path
+license.SetLicense("Aspose.Slides.lic");
 ```
 
-When you call the `SetLicense` method, the string should be same as the name of your license file. For example, you can change the license file name to "*Aspose.Slides.lic.xml*". Then, in your code, you have to pass the new license name *Aspose.Slides.lic.xml* to the `SetLicense` method.
+{{% alert color="warning" %}} 
 
-### **Using a Stream**
-You can load a license from a stream. 
+When you call the `SetLicense` method, the string must have the same name as your license file. For example, you can change the license file name to "*Aspose.Slides.lic.xml*". Then, in your code, you have to pass the new license name *Aspose.Slides.lic.xml* to the `SetLicense` method.

```

Comment:
> the same name as your license file.

Actually it's not just "name", it's a file name or an absolute explicit path (including file name). If this is just a file name, than it can be located in:
- The folder of this assembly.
- The folder of the client's calling assembly.
- The folder of the entry assembly (does not apply for the .NET Compact Framework).